cli/commands/moveContracts.js

```diff
@@ -0,0 +1,26 @@
+
+var fs = require('fs')
+var rimraf = require('rimraf')
+
+let sourceDirectory = '../node_modules/transmute-framework/build/contracts/TransmuteFramework/'
+let destinationDirectory = '../contracts/TransmuteFramework/'
```

Comment:
I could be a dick and say these need to be in a constants / paths file, but given that this is just a cli tool, that might be overkill.